Css/WebScrap.Css.Preprocessing/Readers/CssAttributesLookupReader.cs

```diff
@@ -0,0 +1,71 @@
+using WebScrap.Css.Common.Attributes;
+
+namespace WebScrap.Css.Preprocessing.Readers;
+
+internal ref struct CssAttributesReader
+{
+    private static readonly char[] delimeters = [' ', '>'];
+    private ReadOnlySpan<char> css;
+
+    internal CssAttributesReader(ReadOnlySpan<char> css)
+    {
+        ThrowIfUnsupported(css);
+        this.css = css;
+    }
+
+    private readonly int CurrentTokenIndex
+        => css.LastIndexOfAny(".#");
+
+    public static int Read(ReadOnlySpan<char> css, out CssAttributesLookup attributes)
+    {
+        var tokenDelimeterIndex = css.LastIndexOfAny("> ");
+        if (tokenDelimeterIndex != -1)
+        {
+            return Read(css[tokenDelimeterIndex..][1..], out attributes);
+        }
```

Comment:
Please move this logic to `ctor`.
I assume that every reader clips the css for its own sake.